### DISCUSSION TITLE
Magda installation dependencies

### DISCUSSION THREAD

**Discussion by iturncoffeeintocode** [magda-io/magda#3566] [General]
First off, thank you for sharing this wonderful project.

Referencing to [link](https://github.com/magda-io/magda-brown-bag/blob/main/simple-deployment.md), I am facing issues installing Magda v4.0.0 in an Ubuntu 22.04 Virtualbox VM with the following installed:

- Minikube v1.33.0 (docker 2.29.0 driver used, kubernetes v1.33)
- kubectl v1.33
- helm v3.16.1

I managed to get Magda v2.0.1 up when the following are used:

- ubuntu 20.04
- Minikube v1.28.0 (kubernetes v1.18.0)
- kubectl v1.18.0
- helm 3.16.1

Is there a specific version version of the kubernetes cluster etc that I would need in order to setup Magda v4?

**Comment by t83714** [comment]
Hi @iturncoffeeintocode,

The latest version 4.2.4 requires k8s version higher than: 1.23.0-0

https://github.com/magda-io/magda/blob/3c9f3c2c4e5b68d70315df0c4f063e1f47aa209f/deploy/helm/internal-charts/opensearch/Chart.yaml#L5

If your cluster version is lower than it, the helm should report an error about the version requirements.

Please note, the cluster version here is the server (API) version. To get it, you can use command: `kubectl version` --- this will print both client version and server version.

If you could share the cluster server version and error message, I probably can help to look into more.
